samples/excel/09-events/selection-changed.yaml

```diff
@@ -0,0 +1,116 @@
+name: Selection Changed
+description: Add and remove an event handler on the selection changed event
+host: EXCEL
+api_set: {}
+script:
+    content: |
+        $("#add-event-handler").click(addEventHandler);
+        $("#remove-last-event-handler").click(removeLastEventHandler);
+        $("#remove-all-event-handlers").click(removeAllEventHandlers);
+
+        let lastHandler: OfficeExtension.EventHandlerResult<Excel.Workbook>;
+        let workbook: Excel.Workbook;
+
+        async function addEventHandler() {
+            try {
+                if (workbook !== undefined) {
```

Comment:
FYI:  There is a pending bug fix on my plate that will make this workbook !== undefined not be needed.  I.e., it will make Excel.run be OK with taking in a null/undefined object.  So in a few weeks, we should be able to remove this.